receiver/gitproviderreceiver/internal/scraper/githubscraper/helpers_test.go

```diff
@@ -421,8 +523,16 @@ func TestGetRepos(t *testing.T) {
 			client := graphql.NewClient(server.URL, ghs.client)
 
 			_, count, err := ghs.getRepos(context.Background(), client, "fake query")
+			//sn, count, err := ghs.getRepos(context.Background(), client, "fake query")
 
-			assert.Equal(t, tc.expected, count)
+			//var totalVulnerabilities int
```

Comment:
If we're not using this code let's delete.

Reply:
there's still commented out code here not in use.

Reply:
This was removed this morning, not sure why it showed in your diff.